### PR TITLE
Update ar.json

### PR DESCRIPTION
**Pull request by ali-alhaidary**

### CODE REVIEW

File: packages/tldraw/src/translations/ar.json [hunk 1]
```diff
@@ -1,106 +1,110 @@
...
+    "search": "بحث",
+    "loading": "{dots}تحميل ",
+    "export.background": "تصدير الخلفية",
+    "transparent": "شـفاف",
+    "auto": "ذاتي",
+    "light": "خفيف",
+    "dark": "غامق",
+    "github": "Github",
+    "twitter": "Twitter",
+    "discord": "Discord"
```
Comment (judicaelandria) [inline packages/tldraw/src/translations/ar.json:109]: Why did you add it again? I removed it in your last commit
please remove it

**Review by judicaelandria — CHANGES_REQUESTED** [review]

**Comment by ali-alhaidary** [comment]
@judicaelandria it is already in main.json, I should not be removedt.

**Comment by judicaelandria** [comment]
Removed it from the  main and from ar, should be good!

**Comment by ali-alhaidary** [comment]
@steveruizok  is it really necessary to remove:
` "github": "Github",
  "twitter": "Twitter",
  "discord": "Discord"`
from main.json ?

**Comment by ali-alhaidary** [comment]
close

**Pull request closed without merging**